docs/docs/migrating-from-v1-to-v2.md

```diff
@@ -607,8 +629,32 @@ For example, if you use [`gatsby-plugin-typography`](https://www.gatsbyjs.org/pa
 + export { rhythm, scale, typography as default };
 ```
 
-### createRemoteFileNode
+## For Explorers
 
-The signature for using createRemoteFileNode changed in v2, it now expects a new parameter `createNodeId`.
+### Starting a New Project with v2
 
-[See docs for `createRemoteFileNode`](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-source-filesystem#createremotefilenode)
+Before diving in to the upgrade guide, here's a brief section on starting a new project with Gatsby v2 instead of upgrading an existing project.
```

Comment:
_Before diving in to the upgrade guide_ doesn't work so well now this is at the end of the document

Reply:
yeah i saw that before, updating now!